database/frontend/themes-templates.json

```diff
@@ -19,5 +19,19 @@
         "url": "https://templatemo.com/",
         "category": "frontend",
         "subcategory": "themes-templates"
-    }
+    },
+    {
+        "name": "landing page template",
+        "description": "Simple, free, fully responsive one-page sites for pretty much anything.",
+        "url": "https://carrd.co",
+        "category": "frontend",
+        "subcategory": "themes-templates"
+    },
+    {
+         "name": "Creative-tim",
+        "description": "Simple, free, fully responsive UI Kits, Templates and Dashboards built on top of Bootstrap, Vue.js, React, Angular, Node.js and Laravel.",
```

Comment:
Interesting description @darkweb907, but what does this source different from other websites that offer these templates? I think it would beneficial to add this information to pique developers' interests.

Reply:
Suggest something let's work on it together

Reply:
From what I've seen @darkweb907, this source offers additional resources that teach you the fundamentals of UI/UX. Maybe you can add that there?